backend/app/services/filter_service.rb

```diff
@@ -40,7 +40,34 @@ def sanitize_key(key, table_name)
     end
   end
 
+  def global_search_query(value)
```

Comment:
I think that would be great to add some test for this method in the spec.
The test wrote are maybe not specific enough ?